### ISSUE TITLE
[OSX][TorConnect] AirVPN failing Tor authentication

### ISSUE THREAD

**Issue by rprimus** [AirVPN/airvpn-client#33]
Mon Jun  6 11:43:17 BST 2016

Upgrading to TorBrowser 6.0 breaks AirVPN (due to bug fix):

https://trac.torproject.org/projects/tor/ticket/13252

This fix for this bug changes the cookie file location to under the user's home directory:

``` 
% mdfind -name control_auth_cookie
/Users/rprimus/Library/Application Support/TorBrowser-Data/Tor/control_auth_cookie
```

Line 57 of  ```src/Core/TorControl.cs```:

```c++
^I^I^I^Istring path = "/Applications/TorBrowser.app/TorBrowser/Data/Tor/control_auth_cookie";^I^I^I^I
```

**Comment by evict** [comment]
I hope they will update the client soon. As a workaround I did the following:

```
x:~$ mkdir -p /Applications/TorBrowser.app/TorBrowser/Data/Tor/
x:~$ ln -s /Users/$(whoami)/Library/Application\ Support/TorBrowser-Data/Tor/control_auth_cookie /Applications/TorBrowser.app/TorBrowser/Data/Tor/control_auth_cookie
```

**Comment by rprimus** [comment]
Sat Oct 15 15:55:20 BST 2016

Closing as was fixed in commit 755cd92db8f9c2de7c297a5bafac895ced7c174f
